### PR TITLE
Game page Frontend

### PR DESCRIPTION
**Pull request by nrojo1124**

### CODE REVIEW

File: frontend/src/components/game/PeoplePicker.tsx [hunk 1]
```diff
@@ -27,81 +27,74 @@ const Select: React.FC<PeoplePickerProps> = ({
...
-                    people[2],
-                    `${people[2].name}`,
-                    people[2].id === selectedPerson,
-                    100,
-                    disabled
-                  )}
-                </div>
+            </div>
+          )}
+          {/* index 2: third user*/}
```
Comment (thekevinWHYuan) [inline frontend/src/components/game/PeoplePicker.tsx:72]: I'm getting myself as an opponent in people picker. I don't know if that was the intention or not but I'm assuming this made it so the number of opponents is a fixed number.

File: frontend/src/components/game/PeoplePicker.tsx [hunk 2]
```diff
@@ -27,81 +27,74 @@ const Select: React.FC<PeoplePickerProps> = ({
...
+    <div>
+      <path
+        stroke-linecap="round"
+        stroke-linejoin="round"
+        stroke-width="2"
+        d="M19 13l-7 7-7-7m14-8l-7 7-7-7"
+      />
+      {/* rendering the "enemy" */}
+      <div>
+        {/* index 0 first*/}
```
Comment (thekevinWHYuan) [inline frontend/src/components/game/PeoplePicker.tsx:39]: Maybe map the enemies into some HTML components until there's 3.

File: frontend/src/components/game/PeoplePicker.tsx (hunk 1 above)
Comment (SanTran113) [reply frontend/src/components/game/PeoplePicker.tsx:72]: Yup the number of opponents are hard-coded for now.

**Review by SanTran113 — APPROVED** [review]
Looks good to me!

**Review by Anamooz — APPROVED** [review]

**Review by thekevinWHYuan — APPROVED** [review]
LGTM!

**Pull request merged**